---
layout: page
title: Essential regulatory elements of high-level amplicons as cancer type specific growth dependencies
description: Max-Eder Project 
img: assets/img/DKH_Logo_s.jpg
importance: 1
category: lab
related_publications: dubois2022structural, NatureCancer3.8
---

Most cancer genomes experience DNA amplifications as the most common gain-of-function alterations leading to overexpression of oncogenes. 

Why does this work? 
How does the transcriptional regulation of the oncogenes get hijacked on these amplifications? 
Do they exploit tissue-specific enhancers and transcription factors for oncogene expression and cancer growth? 

We combine whole genome sequencing of hundreds of breast and lung cancers including long-read and epigenetic analyses with validation of our findings using CRISPR-based genome engineering to answer these questions. 

Understanding which regulatory factors are essential for cancer growth could lead to novel tissue-specific gene regulation targeting therapies.

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/ecDNA_FISH.jpg" title="ecDNA FISH" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
    Metaphase FISH image of cells with normal karyotype (left) and brain cancer cells with a high-level amplicon of MYCN and ID2 on extrachromosomal DNA (right) – a phenomenon only observed in malignant cells. (image from Dubois F et al. Nature Cancer. 2022.) 
</div>
The coding sequences of the oncogenes are only a small fraction of the amplified DNA. Why are the cancer cells keeping 50+ copies of the rest? Are there essential regulatory elements in these co-amplified regions?
<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/MYCN_ID2ecDNARec.jpg" title="MYCN-ID2 ecDNA" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
Reconstruction of an MYCN-ID2 extrachromosomal DNA (ecDNA) amplicon. The plot shows two loci on chromosome 2 with the positions of the coding sequences of the MYCN and ID2 genes highlighted in blue and orange respectively. The larger red boxes show co-amplified regions that could contain essential regulatory elements. (from Dubois F et al. Nature Cancer. 2022.) 
</div>

To answer these questions, we will analyze several hundred breast and lung cancer tumor samples by whole genome sequencing (WGS), including with long-read technology.

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/CNprofile.jpg" title="CN profile" class="img-fluid rounded z-depth-1" %}
    </div>
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/longreads_lbld.jpg" title="long reads" class="img-fluid rounded z-depth-1" %}
    </div>
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/ecDNAgenericRec.jpg" title="ecDNA structures" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
    (Left) Copy number profile resulting from WGS. (Center) Long-read technology will allow us to detect SVs in the cancer genome (top track) and methylation (middle track) from which we can infer the positions of enhancers and transcription start sites (bottom track). Combing all this data, we can reconstruct the topology of the amplicon (Right)
</div>

<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/ATAC_s.jpg" title="ATAC profiles" class="img-fluid rounded z-depth-1" %}
    </div>
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/HiC.jpg" title="HiC" class="img-fluid rounded z-depth-1" %}
    </div>
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/ecDNA_links.jpg" title="ecDNA links" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
    Using differences in the epigenetic profiles (left) between tumors with high (top 2 tracks) and low (bottom 2 tracks) expression of the oncogene and HiC to measure 3D genome interactions (center), we will infer which enhancers on the amplicon activate oncogene transcription. By analyzing amplicons of the same oncogene in both non-small cell lung cancer (NSCLC) and breast cancer (BRCA), we can test if there is tissue type specific enhancer usage (right).
</div>

<div class="row justify-content-sm-center">
    <div class="col-sm-8 mt-3 mt-md-0">
        {% include figure.html path="assets/img/CRISPR.jpg" title="example image" class="img-fluid rounded z-depth-1" %}
    </div>
    <div class="col-sm-4 mt-3 mt-md-0">
        {% include figure.html path="assets/img/prelim.jpg" title="example image" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
    We will validate which regulatory elements are essential for oncogene expression and cancer cell growth using CRISPR-based genome engineering (left). We have already identified dozens of suitable cell lines of both cancer types that show amplifications of the same oncogenes to test our hypotheses (right).
</div>

We are part of a multi-institutional network of collaborators combining efforts to understand cancer genomes.
<div class="row">
    <div class="col-sm mt-3 mt-md-0">
        {% include figure.html path="assets/img/colabs.jpg" title="network" class="img-fluid rounded z-depth-1" %}
    </div>
</div>
<div class="caption">
    Our collaborators at Charité and other leading cancer centers.   
</div>